core/src/main/java/gov/nist/secauto/metaschema/core/model/constraint/LoggingConstraintValidationHandler.java

```diff
@@ -101,21 +101,39 @@ private void logConstraint(
   public void handleCardinalityMinimumViolation(
       @NonNull ICardinalityConstraint constraint,
       @NonNull INodeItem node,
-      @NonNull ISequence<? extends INodeItem> targets) {
+      @NonNull ISequence<? extends INodeItem> targets,
+      @NonNull DynamicContext dynamicContext) {
     Level level = constraint.getLevel();
     if (isLogged(level)) {
-      logConstraint(level, node, newCardinalityMinimumViolationMessage(constraint, node, targets), null);
+      logConstraint(
+          level,
+          node,
+          newCardinalityMinimumViolationMessage(
+              constraint,
+              node,
+              targets,
+              dynamicContext),
+          null);
```

Comment:
Very minor: I know only I really like this style, and others code including me code with IDE for free in mind, but I get worried about undeclared consts (even `null`) as arg or parameter in code without explanation. In some places I do enums that bind to a value like that to make it clear what's going on, but with `null` that is probably overkill and not good, even if possible.

Anyway, comment here?

```suggestion
          null); // Null because there is no exeception, a Throwable cause.
```